charts/alfresco-sync-service/templates/_helpers.tpl

```diff
@@ -18,3 +18,11 @@ chart: {{ .Chart.Name }}
 {{ include "syncservice.selectorLabels" . }}
 heritage: {{ .Release.Service }}
 {{- end }}
+
+{{- define "syncservice.repository" -}}
+{{- if .Values.repository.nameOverride }}
+{{- printf "%s-%s" .Release.Name .Values.repository.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- .Values.repository.host }}
+{{- end }}
+{{- end -}}
```

Comment:
we will move this once repository get its own chart I imagine

Reply:
I would like to yes but it mostly depends on what the acs-deployment charts becomes eventually